brave/brave-core
# Adds retries for unverified publishers

Resolves https://github.com/brave/brave-browser/issues/3741

UI PR: https://github.com/brave/brave-ui/pull/482

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:
- enable rewards
- close browser
- go to publisher_list and change second value for `3zsistemi.si` to false from `["3zsistemi.si",true,false....` to `["3zsistemi.si",false,false....`
- start the browser
- add tip to unverified site
- add tip for 10, 10, 5 and 10 to 3zsistemi.si
- add another tip to unverified site
- go to 3zsistemi.si, open panel and click check again
- go to settings page and observe the page after couple of minutes 3 pending contributions (10, 10, 5) should be processed and you should have two notifications (new publisher verified and out of funds)
- open pending contributions dialog and make sure that you see 3 tips (2 to your unverified site and one to 3zsistemi for 10)

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
